R/detect.R

```diff
@@ -1,15 +1,16 @@
 #' Detect the presence/absence of a match
 #'
-#' `str_detect()` returns a logical vector `TRUE` if `pattern` is found within
-#' each element of `string` or a `FALSE` if not. It's equivalent
+#' `str_detect()` returns a logical vector with `TRUE` for each element of
+#' `string` that matches `pattern` and `FALSE` otherwise. It's equivalent to
```

Comment:
This is the change that motivates the PR. The original is missing a few words or punctuation? So I took a stab at rewording the description to deal with both.

I got here because this documentation is also inlined into the `man.Rmd` chapter of R Packages, so it needs some kind grammatical fix. Either mine or something else.